cranecloud/mkdocs.yml

```diff
@@ -3,7 +3,13 @@ nav:
   - quickstart:
       - quickstart: quickstart/quickstart.md
   - Projects: project.md
-  - Applications: apps.md
+  - Applications:
+     - Application Containerization: applications/appContainerization.md
+     - Application Creation/Deployment: applications/deployApp.md
+     - Deploy Application without Database Support: applications/deployWithoutDB.md
+     - Deploy an app Image from Docker with Database Support: applications/dockerWithDB.md
+     - Deploy an app Image from GCR with Database Support: applications/gcrWithDB.md
+     - Delet deployed application: applications/deleteApp.md
```

Comment:
Typo detected `- Delet`

Reply:
@MutegekiHenry this has been fixed, as shown in the screenshot